editoast/src/client/import_rolling_stock.rs

```diff
@@ -70,162 +76,251 @@ pub async fn import_rolling_stock(
+            assert!(raise_pantograph_time.is_some());
+        }
     }
 
-    #[rstest]
-    async fn import_electric_rs_with_startup_and_panto_values() {
-        // GIVEN
-        let db_pool = DbConnectionPoolV2::for_tests();
-        let rolling_stock_name =
-            "fast_rolling_stock_import_electric_rs_with_startup_and_panto_values";
-        let electric_rolling_stock = get_fast_rolling_stock_schema(rolling_stock_name);
-        let file = generate_temp_file(&electric_rolling_stock);
-        let args = ImportRollingStockArgs {
-            rolling_stock_path: vec![file.path().into()],
-        };
+    mod towed_rolling_stock {
+        use super::*;
+
+        use crate::client::generate_temp_file;
+
+        use editoast_models::DbConnectionPoolV2;
+        use rstest::rstest;
+
+        #[rstest]
+        async fn import_towed_rolling_stock_ko_file_not_found() {
+            // GIVEN
+            let db_pool = DbConnectionPoolV2::for_tests();
+            let args = ImportRollingStockArgs {
+                rolling_stock_path: vec!["non/existing/railjson/file/location".into()],
+            };
+
+            // WHEN
+            let result = import_towed_rolling_stock(args, db_pool.into()).await;
+
+            // THEN
+            assert!(result.is_err())
+        }
+
+        #[rstest]
+        async fn import_valid_towed_rolling_stock() {
+            // GIVEN
+            let db_pool = DbConnectionPoolV2::for_tests();
+            let towed_rolling_stock_name = "towed";
+            let mut towed_rolling_stock_form: TowedRollingStock =
+                serde_json::from_str(include_str!("../tests/example_towed_rolling_stock_1.json"))
+                    .expect("Unable to parse");
+            towed_rolling_stock_form.name = towed_rolling_stock_name.to_string();
+
+            let file = generate_temp_file(&towed_rolling_stock_form);
+            let args = ImportRollingStockArgs {
+                rolling_stock_path: vec![file.path().into()],
+            };
+
+            // WHEN
+            let result = import_towed_rolling_stock(args, db_pool.clone().into()).await;
 
-        // WHEN
-        let result = import_rolling_stock(args, db_pool.clone().into()).await;
-
-        // THEN
-        assert!(result.is_ok(), "import should succeed");
-        use crate::models::Retrieve;
-        let created_rs =
-            RollingStockModel::retrieve(&mut db_pool.get_ok(), rolling_stock_name.to_string())
-                .await
-                .expect("Failed to retrieve rolling stock")
-                .unwrap();
-        let RollingStockModel {
-            electrical_power_startup_time,
-            raise_pantograph_time,
-            ..
-        } = created_rs;
-        assert!(electrical_power_startup_time.is_some());
-        assert!(raise_pantograph_time.is_some());
+            // THEN
+            assert!(result.is_ok());
+            use crate::models::Retrieve;
```

Comment:
I think all `use ..::Retrieve` are unnecessary since the `prelude` is already imported at top-level and you `use super::*` in every test module.

Reply:
Done in [f582efe](https://github.com/OpenRailAssociation/osrd/pull/9580/commits/f582efe1d9ae87256116f815baf1b6b94455ff6c).